lib/fsm.c

```diff
@@ -231,6 +232,9 @@ static int expandRegular(rpmfi fi, const char *dest, rpmpsm psm, int nodigest, i
 exit:
     if (wfd) {
 	int myerrno = errno;
+        if (rpmExpandNumeric("%{force_fsync_on_close}")) {
```

Comment:
`oneshot` is processed in `processSourceFiles` which won't apply here... as I imagine everythign in build/ is ... where would be an appropriate place to check for this and set it once?